utils/downloader.py

```diff
@@ -0,0 +1,29 @@
+# File description:
+# Simple utilities to download and save a file with progress bar.
+import os
+from urllib.error import URLError
+from urllib.request import urlretrieve
+
+def show_progress(blk_num, blk_sz, tot_sz):
+    percentage = 100. * blk_num * blk_sz / tot_sz
+    print('Progress: %.1f %%' % percentage, end='\r', flush=True)
+
```

Comment:
[Two blank lines between top-level definitions, be they function or class definitions](https://google.github.io/styleguide/pyguide.html#35-blank-lines)

Reply:
Got it.

For future guidelines, is there a Python code style you follow and please refer me to the document so that I will try to remember. It is also good to create a CONTRIBUTION file in your project root for potential contributors to follow.

Reply:
I have created another PR (#6) for this purpose.